pipeline/dbt/models/staging/sources/imilo/stg_imilo__offres.sql

```diff
@@ -0,0 +1,35 @@
+WITH source AS (
+    {{ stg_source_header('imilo', 'offres') }}
+),
+
+structures_offres AS (
+    SELECT
+        CAST((data -> 'structures_offres' ->> 'offre_id') AS INTEGER)         AS "id_offre",
+        CAST((data -> 'structures_offres' ->> 'missionlocale_id') AS INTEGER) AS "id_structure",
+        CONCAT(
+            (data -> 'structures_offres' ->> 'offre_id'),
+            '_',
+            (data -> 'structures_offres' ->> 'missionlocale_id')
+        )                                                                     AS "offre_structure_id"
+    FROM {{ source('imilo', 'structures_offres') }}
+),
```

Comment:
mieux vaut un model dédié pour `structures_offres` qui matchent au plus près ce qu'ils nous exposent, avec des tests propres

le but de staging c'est de mettre les données au format tabulaire et de faire les nettoyages incontournables, sans changer le model de données à ce stade

le fait de faire le "produit" entre structures et offres, c'est déjà plus ce qu'on considère du mapping

Reply:
Bien noté, j'ai fait le mapping dans intermediate du coup :)